### PR TITLE
Added InsightObject updating functionality

### PR DESCRIPTION
**Pull request by maximbetin**
You can now edit objects in the same fashion as with creating them.

### CODE REVIEW

File: jira_insight/insight.py [hunk 1]
```diff
@@ -92,6 +98,21 @@ def __init__(self, insight, object_id, object_json=None):
...
 
+    def update_object(self, attributes: dict):
+        attributes_json = []
+        for attribute_id, value in attributes.items():
+            entry = {
+                "objectTypeAttributeId": attribute_id,
+                "objectAttributeValues": [{"value": value}],
+            }
+            attributes_json.append(entry)
+        request_body = {"attributes": attributes_json}
```
Comment (schuppentier) [inline jira_insight/insight.py:109]: According to the [Docs](https://insight-javadoc.riada.io/insight-javadoc-8.6/insight-rest/#object__id__put), Insight requires the objectTypeId to be set when updating objects. So this needs to be added here, we could get it from `self.object_json` directly or we could extract the id in __init__.

**Review by schuppentier — CHANGES_REQUESTED** [review]
Thanks for the PR! There is one small thing with the missing objectTypeId, but else it looks good.

File: jira_insight/insight.py (hunk 1 above)
Comment (maximbetin) [reply jira_insight/insight.py:110]: I believe this should do it, I chose the `self.object_json` route.

Comment (schuppentier) [reply jira_insight/insight.py:110]: Looks good, thank you!

**Pull request merged**